src/theme/index.ts

```diff
@@ -34,6 +34,7 @@ import Radio from "./components/radio";
 import RadioGroup from "./components/radioGroup";
 import SearchBar from "./components/searchBar";
 import { Skeleton, SkeletonLoader } from "./components/skeletonLoader";
+import CustomSlider from "./components/slider";
```

Comment:
Why `CustomSlider` and not just `Slider`?  I recall that other components started with. `Custom` as part of the name, but those names were eventually updated with a more standard name.

Reply:
Because `Slider` is already generated by Chakra and having the same name isn't _wrong_, but it makes it difficult to update or extend certain styles. I had a trouble overriding some styles but if given a different name, there's no issue.